ng2-components/ng2-activiti-form/src/services/form.service.ts

```diff
@@ -47,6 +47,23 @@ export class FormService {
                 private logService: LogService) {
     }
 
+    parseForm(json: any, data?: FormValues, readOnly: boolean = false): FormModel {
```

Comment:
Can not retrun an empty form instead of null?I would like to prefer to not propagate null in the code